modules/@apostrophecms/piece-type/ui/apos/components/AposDocsManager.vue

```diff
@@ -240,7 +264,6 @@ export default {
         }
       });
     }
-    apos.bus.$on('content-changed', this.getPieces);
```

Comment:
@ValJed Was this change intentional? It's breaking the manager refresh and doesn't seem directly related to the PR.

Reply:
I added it back in the feature branch for now, but let me know if there is another way to handle this.